kratos/sources/kratos_parameters.cpp

```diff
@@ -202,7 +202,7 @@ const std::string Parameters::const_iterator_adaptor::name()
 
 Parameters::Parameters()
 {
-    mpRoot = Kratos::make_shared<nlohmann::json>(nlohmann::json::parse( "{}" ));
+    mpRoot = Kratos::make_shared<nlohmann::json>(nlohmann::json::parse( "{}", nullptr, true, true));
```

Comment:
they broke their API? I didn't read this in the release notes

Reply:
Yep, I just follow the documentation (I have only changed the last argument from false to true)

Reply:
what does this mean? (asking so that the change is documented in this PR for future reference)

Reply:
https://github.com/nlohmann/json/blob/d34771cafc87b358ba421faca28facc7f8080174/single_include/nlohmann/json.hpp#L23036

Reply:
ok so in order to make use of the comments (which are disabled by default) we need to pass those parameters

Reply:
Basically because is the last argument